### PR TITLE
Added State and Transition Data Tables for ischemic stroke model

### PR DESCRIPTION
**Pull request by SantoniS**
According to GBD 2017, stroke cases are considered acute from the day of incidence of a first-ever stroke through day 28 following the event. Chronic stroke includes the sequelae of an acute stroke AND all recurrent stroke events. Stroke cases are considered chronic beginning 28 days following the occurrence of an event. Chronic stroke includes the sequelae of an acute stroke AND all recurrent stroke events.

Given this information, I would really appreciate someone to review if I assigned ids correctly in the data and transition tables. Thank you!

### CODE REVIEW

File: docs/source/models/gbd2017/causes/ischemic_stroke/index.rst [hunk 1]
```diff
@@ -48,15 +80,150 @@ Model Assumptions and Limitations
...
+   :widths: 1, 5, 20
+   :header-rows: 1
+
+   * - State
+     - State Name
+     - Definition
+   * - A
+     - **A**\ cute Ischemic Stroke
+     - Simulant that is in duration-based period starting day of incidence of a first-ever stroke through day 28 following the event
+   * - C
```
Comment (aflaxman) [inline docs/source/models/gbd2017/causes/ischemic_stroke/index.rst]: Consider changing *C*hroinc to *P*ost for stroke, analogous to what we discussed for IHD yesterday.  Also include susceptible in this list

File: docs/source/models/gbd2017/causes/ischemic_stroke/index.rst [hunk 2]
```diff
@@ -48,15 +80,150 @@ Model Assumptions and Limitations
...
+     - disability weight
+     - :math:`\displaystyle{\sum_{s\in \text{sequelae in c_495}}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
+     - = average disability weight over all sequelae
+   * - All A
+     - cause-specific mortality rate
+     - :math:`\frac{\text{deaths_s6116}}{\text{population}}`
+     -
+   * - All C
+     - cause-specific mortality rate
+     - :math:`\frac{\text{deaths_s6248}}{\text{population}}`
```
Comment (aflaxman) [inline docs/source/models/gbd2017/causes/ischemic_stroke/index.rst]: update analogously to changes we discussed for IHD yesterday

File: docs/source/models/gbd2017/causes/ischemic_stroke/index.rst [hunk 3]
```diff
@@ -48,15 +80,150 @@ Model Assumptions and Limitations
...
+     - 
+   * - duration-based
+     - A 
+     - C 
+     - 28-day duration in acute state then progress
+     - 
+   * - i
+     - C
+     - A 
+     - incidence_c495
```
Comment (aflaxman) [inline docs/source/models/gbd2017/causes/ischemic_stroke/index.rst]: update analogously to changes we discussed for IHD yesterday

File: docs/source/models/gbd2017/causes/ischemic_stroke/index.rst [hunk 4]
```diff
@@ -48,15 +80,150 @@ Model Assumptions and Limitations
...
+     - gbd_mapping
+     - List of 11 sequelae for ischemic stroke
+     -
+   * - prevalence_s{`sid`}
+     - como
+     - Prevalence of sequela with id `sid`
+     - Confirm if needed, as GBD 2017 states that chronic IS includes ALL sequelae of IS
+   * - disability_weight_s{`sid`}
+     - YLD Appendix
+     - Disability weight of sequela with id `sid`
```
Comment (aflaxman) [inline docs/source/models/gbd2017/causes/ischemic_stroke/index.rst]: as above

**Comment by SantoniS** [comment]
Thank you for your very helpful feedback yesterday, @aflaxman  and @collijk!

File: docs/source/gbd2017_models/causes/ischemic_stroke/index.rst [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+     - excess mortality rate (emr)
+     - emr_m9310
+     - 
+   * - S
+     - excess mortality rate (emr)
+     - 0
+     - This is not applicable
+   * - P
+     - disability weight
+     - :math:`\displaystyle{\sum_{s\in \text{391}}}^{395} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}` + disability_weight_s946 :math:`\times` prevalence_s946
```
Comment (yongqx) [inline docs/source/gbd2017_models/causes/ischemic_stroke/index.rst]: for prevalence weighted average of disability weight of Post IS state, my understanding of equation is:
sum(dw_s * prev_s) / sum(prev_s) where s = 391 to 395 plus 946

File: docs/source/gbd2017_models/causes/ischemic_stroke/index.rst [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+     - = average disability weight over all sequelae in the post ischemic stroke state
+   * - A
+     - disability weight
+     - :math:`\displaystyle{\sum_{s\in \text{386}}}^{380} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
```
Comment (yongqx) [inline docs/source/gbd2017_models/causes/ischemic_stroke/index.rst]: same logic disability of Acute IS state = sum(dw_s * prev_s) / sum(prev_s) where s = 386 to 390.

File: docs/source/gbd2017_models/causes/ischemic_stroke/index.rst (hunk 5 above)
Comment (SantoniS) [reply docs/source/gbd2017_models/causes/ischemic_stroke/index.rst]: Yes, that is correct.

File: docs/source/gbd2017_models/causes/ischemic_stroke/index.rst (hunk 6 above)
Comment (SantoniS) [reply docs/source/gbd2017_models/causes/ischemic_stroke/index.rst]: Yes, correct! I noticed that the equation above is incorrect as it says s= 386 to 380. That was a typo, and fixed the upper limit in the sum to be 390.

**Comment by SantoniS** [comment]
The commit titled 'Corrected Post-MI IHD disability weight formula.' is incorrectly titled. It should say 'Corrected Post State disability weight formula.' It is correct data according to Ischemic Stroke model but my mind made a human error and added the words 'MI IHD', as I am used to typing it. My mistake!

**Comment by SantoniS** [comment]
@yongqx  Based on answers above, do you think this PR is ready to be merged to master?

**Comment by yongqx** [comment]
> @yongqx Based on answers above, do you think this PR is ready to be merged to master?

I just approved it!

**Pull request merged**